### DISCUSSION TITLE
Crash Fix on ShowDebug ALS.Curves console commands

### DISCUSSION THREAD

**Discussion by emaovo** [Sixze/ALS-Refactored#93] [Show and tell]
hi all, 

I often had crashes when typing console commands _ShowDebug ALS.State or other ALS. functions_, due to the DebugData chain not initialized. I added the call to Super and now it works. 

Just wanted to let you know in case you want to integrate it. 
All the best

```
void FAlsAnimNode_CurvesBlend::GatherDebugData(FNodeDebugData& DebugData)
{
	DECLARE_SCOPE_HIERARCHICAL_COUNTER_ANIMNODE(GatherDebugData)
	Super::GatherDebugData(DebugData);// add at least one entry in the chain to prevent crash in BranchFlow when NodeChain is empty
	SourcePose.GatherDebugData(DebugData.BranchFlow(1.0f));
	CurvesPose.GatherDebugData(DebugData.BranchFlow(BlendAmount));
}
```

**Comment by Sixze** [comment]
Fixed in 7278d45e43d51e979566bbd5670f8a9400d32e10.
